src/main/java/seedu/address/logic/parser/MarkCommandParser.java

```diff
@@ -26,7 +25,7 @@ public class MarkCommandParser implements Parser<MarkCommand> {
     public MarkCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_WEEK, PREFIX_INDICES);
-        if (!arePrefixesPresent(argMultimap, PREFIX_WEEK, PREFIX_INDICES)
+        if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_WEEK, PREFIX_INDICES)
```

Comment:
Same here, I have modified the parse function for MarkCommand so we check for PREFIX_PRESENT and PREFIX_ABSENT instead.